cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
This class is new to me. It was added as part of the content library work:

https://github.com/edx/edx-platform/pull/11611

It is interesting that of the two other services, ``DjangoXBlockUserService`` is added both here and to the preview module system, while ``StudioPermissionsService`` is only added here. I'm not sure which is the better model to follow. I don't love that we now have to register services in two different places. @cahrens, what do you think?